frappe/core/doctype/deleted_document/deleted_document_list.js

```diff
@@ -0,0 +1,32 @@
+frappe.listview_settings["Deleted Document"] = {
+	onload: function (doclist) {
+		const action = () => {
+			const selected_docs = doclist.get_checked_items();
+			if (selected_docs.length > 0) {
+				let docnames = selected_docs.map((doc) => {
+					return doc.name;
+				});
+				frappe.call({
+					method: "frappe.core.doctype.deleted_document.deleted_document.bulk_restore",
+					args: { "docnames": docnames },
+					callback: function (r) {
+						if (r.message) {
+							let num = r.message.length;
+							let message;
+							if (num === 1) {
+								message = "{0} Document was Restored";
+							} else {
+								message = "{0} Documents were Restored";
+							}
+							frappe.msgprint(__(message, [num]));
```

Comment:
```suggestion
							if (num === 0) {
								message = __("No Document were Restored");
							} else if (num === 1) {
								message = __("Document was Restored");
							} else {
								message = __("{0} Documents were Restored", [num]);
							}
							frappe.msgprint(message);
```

Having the message itself wrapped in `__()` makes it possible to scrape it from the codebase and add it as a translation candidate for the future.